### DISCUSSION TITLE
How to Load and Use Trained Model from .tar Files for Calculations in CHGNet?

### DISCUSSION THREAD

**Discussion by ShuaishuaiYuan** [CederGroupHub/chgnet#158] [Q&A] [answered]
In the `chgnet/examples/fine_tuning.ipynb`, it is mentioned that "After training, the trained model can be found in the directory of today's date." I see there are some *.tar files generated after training but how can I load these files and use the updated model to perform calculations, such as relaxation? Could you please provide some example scripts? Thank you!!

**Comment by chtchelkatchev** [comment, marked as answer]
E.g., 
chgnet = CHGNet.from_file("../05-29-2024/epoch599_e0_f13_s5_m1.pth.tar")
